packages/duckdb/index.cjs

```diff
@@ -121,7 +125,13 @@ module.exports.getRunner = async (opts, directory) => {
 module.exports.testConnection = async (opts, directory) => {
 	const r = await runQuery('SELECT 1;', { ...opts, filename: path.join(directory, opts.filename) })
 		.then(() => true)
-		.catch((e) => ({ reason: e.message ?? 'File not found' }));
+		.catch((e) => {
+			if (typeof e === 'string' && Boolean(e)) {
```

Comment:
Is the `Boolean(e)` only meant to catch empty strings? Would be clearer as an `e !== ''` imo

Reply:
```suggestion
			if (typeof e === 'string' && e !== '') {
```